shared/js/ui/templates/search.es6.js

```diff
@@ -8,8 +8,7 @@ module.exports = function () {
         <input type="text" autocomplete="off" placeholder="Search DuckDuckGo"
             name="q" class="search-form__input js-search-input"
             value="${this.model.searchText}" />
-        <input class="search-form__go js-search-go" tabindex="2" value="" type="button" />
-        <input type="submit" class="search-form__submit" />
```

Comment:
This threw me. There were two submit buttons (a hidden submit one and a non-hidden plain button) for some reason. I don't understand why that was the case, but I accidentally had the label on the second hidden one.

I removed it. It seemed unnecessary.